.github/workflows/markdown-quality.yml

```
@@ -0,0 +1,158 @@
+      
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+        cache: 'npm'
+        
+    - name: Install markdownlint
+      run: npm install -g markdownlint-cli2
+      
+    - name: Cache quality baseline
+      uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)